.github/ci_templates/build.yml

```
@@ -15,7 +15,7 @@ steps:
   - name: Run go generate
     run: |
       {!{/* TODO replace with 'make generate' when make become available on runners */}!}
-      (cd tools && go generate)
+      (ln -s "$(pwd)" /deckhouse && cd tools && go generate)
```

```suggestion
      (cd tools && go generate)
```
It should exist in your environment; it is explained in the development guide. It may prevent the script from running on laptops with MacOS onboard.